internal/driver/fetch_test.go

```diff
@@ -206,6 +206,138 @@ func TestFetch(t *testing.T) {
 	}
 }
 
+func TestFetchWithBase(t *testing.T) {
+	const path = "testdata/"
+	// set up testcases
+	type testcase struct {
+		sources         []string
+		bases           []string
+		normalize       bool
+		testDescription string
+		expectSamples   bool
+	}
+
+	testcases := []testcase{
+		{
+			[]string{
+				path + "cppbench.cpu",
+			},
+			[]string{
+				path + "cppbench.cpu",
+			},
+			false,
+			"not normalized, base is same as non-base",
```

Comment:
Maybe word it as "not normalized, base is same as source profile"? We call these "source profiles" in fetch.go.